HelpSource/Classes/SimpleNumber.schelp

```diff
@@ -992,14 +992,15 @@ Returns a random number from a uniform distribution with the range [-this, +this
 i.e. between -this and +this (including -this and +this).
 
 method:: rrand
-Returns a random number from a uniform distribution with the range between code::this:: and code::arg::.
-argument::arg
+Returns a random number from a uniform distribution with the range between code::this:: and code::aNumber::.
+argument::aNumber
 The other (upper or lower) limit of the range.
+argument::adverb
 
-discussion:: 
-code::arg:: can be lower than code::this::.
-If either code::this:: or code::arg:: is a link::Classes/Float::, the code::arg:: bound will be excluded from the range.
-If both code::this:: and code::arg:: are link::Classes/Integer::, both bounds are included.
+discussion::
+code::aNumber:: can be lower than code::this::.
+If either code::this:: or code::aNumber:: is a link::Classes/Float::, the code::aNumber:: bound will be excluded from the range.
```

Comment:
Can we add the return types here as well. If either is a float a float will be returned. If both are integers an integer will be returned.

Reply:
As follows?

```
If either code::this:: or code::aNumber:: is a link::Classes/Float::, the code::aNumber:: bound will be excluded from the range, and sclang returns an integer.
If both code::this:: and code::aNumber:: are link::Classes/Integer::, both bounds are included, and sclang returns a float.
```

Reply:
Wonderful. Thanks!